Linch's test edits

InALunch opened the pull request.

InALunch committed:
Rename Pricing to About Us

InALunch committed:
Fix issue with escape apostrophe

akrolsmir commented on src/components/NavBar.tsx:
```diff
@@ -16,7 +16,7 @@ const sections = [
   },
   { id: 'screencasts', title: 'Screencasts' },
   { id: 'resources', title: 'Resources' },
-  { id: 'pricing', title: 'Pricing' },
```
BTW, later it would be nice to change the id: to say "about-us" too, because that will be shown in the url (like "https://asteroid.org/#about-us"

InALunch committed:
Update introduction

Change introduction to be a bit more consistent with the planned rest of text.

InALunch committed:
Tidy language in FAQ, remove details of Operation D*

InALunch committed:
Make smoother language for the FAQ

eg spell out the "something somethings." And make it sound more like a real FAQ, not a live interview.

InALunch committed:
Change title

Open Asteroid Impact -> Introducing Open Asteroid Impact, to more closely mimic https://openai.com/blog/introducing-openai

akrolsmir committed:
Merge branch 'main' into Linch's-branch
